packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -63,6 +65,8 @@ export class PincodeSet extends React.Component<Props, State> {
   navigateToNextScreen = () => {
     if (this.props.choseToRestoreAccount) {
       navigate(Screens.ImportWallet)
+    } else if (this.props.hideVerification || !this.props.route.params?.komenciAvailable) {
```

Comment:
Should we have this check on `ImportWallet` as well?

Reply:
Good call out but I think no. It would be the "correct" thing to do but ultimately I don't think it's worth it because:
1. the overwhelming majority of users will be going through the "new account" flow. we do another check on the VerificationEducationScreen anyways so the worst case is they see a message there instead of skipping the step entirely
2. not really worth increasing the surface area of the hotfix given the marginal benefit. the navigation happens in the saga, which would require us to pass a param through the action, and i think we should keep changes as minimal as possible

Reply:
That makes sense to me